quiz/readme.md

```diff
@@ -13,3 +13,126 @@
 2. Copy `envs/postgres.default.env` to `envs/postgres.env` and fill environmental variables.
 3. Copy `envs/backend.default.env` to `envs/backend.env` and fill environmental variables.
 4. Start services `docker compose -f docker-compose.prod.yaml up --build`.
+
+
+## Run Tests
+1. Run `docker compose exec <docker_service_name> pytest` or `docker compose exec <docker_service_name> poetry run pytest`.
+
+
+## Example of using logging
+```python
+import logging
+
+logger = logging.getLogger('django')
+
+logger.info("Info message")
+logger.debug("Debug message")
+logger.warning("Warning message")
+```
+
+## Building / Updating Documentation
+
+1. Open a shell inside the container: `docker compose exec <docker_service_name> sh`.
+2. Navigate to the docs folder: `cd docs`.
+3. To build the documentation, type: `sphinx-build -b html . _build`.
+   - Find the documentation in the docs/_build/html folder.
+   - To view the documentation, open the index.html file in a browser.
```

Comment:
Trzeba dodać parę kroków tutaj bo nie wystarczy tylko odpalić `make html`. Po każdym dodaniu nowej apki trzeba zrobić update plików rst. Tę komendę trzeba odpalić w folderze `backend`  `sphinx-apidoc -o docs <nazwa modułu>/`. To generuje pliki rst dla modułów, **ważne** pliki generują się dla każdego folderu z`__init__.py` Potem jak już masz wygenerowany plik modules.rst to trzeba dodać apkę do `.. toctree::` i dopiero wtedy można odpalić `make html`. Jeżu z jabłkiem, nie wiem czy to ma sens co tu napisałam w każdym razie zwykłe `make html` generuje dokumentację tylko jeżli miałeś wcześniej pliki .rst dla danego modułu😅.  A żeby zmienił się `docs/_build/index.html` to trzeba na nowo zbuildować dokumentację.

Albo coś źle robię😅

Do tego można dodać ścieżkę do pliku z html, który updateuje komenda `make html`:

> docs/_build/html/index.html

Reply:
done